config.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+)
+
+// Configs stores the step's inputs
+type Configs struct {
+	JSONKeyPath             stepconf.Secret `env:"service_account_json_key_path,required"`
+	PackageName             string          `env:"package_name,required"`
+	AppPath                 string          `env:"app_path,required"`
+	ExpansionfilePath       string          `env:"expansionfile_path"`
+	Track                   string          `env:"track,required"`
+	UserFraction            string          `env:"user_fraction,opt[0.05,0.1,0.2,0.5]"`
+	WhatsnewsDir            string          `env:"whatsnews_dir"`
+	MappingFile             string          `env:"mapping_file"`
+	UntrackBlockingVersions string          `env:"untrack_blocking_versions,opt[true,false]"`
+
+	// Deprecated
+	ApkPath string `env:"apk_path"`
+}
+
+// validate validates the Configs.
+func (c Configs) validate() error {
+	if err := c.validateJSONKeyPath(); err != nil {
+		return err
+	}
+
+	if err := c.validateWhatsnewsDir(); err != nil {
+		return err
+	}
+
+	if err := c.validateMappingFile(); err != nil {
+		return err
+	}
+
+	return c.validateApps()
+}
+
+// validateJSONKeyPath validates if service_account_json_key_path input value exists if defined and has file:// URL scheme.
+func (c Configs) validateJSONKeyPath() error {
+	if !strings.HasPrefix(string(c.JSONKeyPath), "file://") {
+		return nil
+	}
+
+	pth := strings.TrimPrefix(string(c.JSONKeyPath), "file://")
+	if exist, err := pathutil.IsPathExists(pth); err != nil {
+		return fmt.Errorf("failed to check if json key path exist at: %s, error: %s", pth, err)
+	} else if !exist {
+		return errors.New("json key path not exist at: " + pth)
+	}
+	return nil
+}
+
+// validateWhatsnewsDir validates if whatsnews_dir input value exists if provided.
+func (c Configs) validateWhatsnewsDir() error {
+	if c.WhatsnewsDir == "" {
+		return nil
+	}
+
+	if exist, err := pathutil.IsDirExists(c.WhatsnewsDir); err != nil {
+		return fmt.Errorf("failed to check if what's new directory exist at: %s, error: %s", c.WhatsnewsDir, err)
+	} else if !exist {
+		return errors.New("what's new directory not exist at: " + c.WhatsnewsDir)
+	}
+	return nil
+}
+
+// validateMappingFile validates if mapping_file input value exists if provided.
+func (c Configs) validateMappingFile() error {
+	if c.MappingFile == "" {
+		return nil
+	}
+
+	if exist, err := pathutil.IsPathExists(c.MappingFile); err != nil {
+		return fmt.Errorf("Failed to check if mapping file exist at: %s, error: %s", c.MappingFile, err)
+	} else if !exist {
+		return errors.New("mapping file not exist at: " + c.MappingFile)
+	}
+	return nil
+}
+
+// Apps returns the app to deploy, by prefering .aab files.
```

Comment:
[...] returns the app paths to deploy [...] -> The method name `Apps` is the one we want to use? Also the later client side usage shows that might be a different name would be better: `appPaths := configs.Apps()`. Wdyt about `AppPaths` ? Also please do not export any function or method if it is strictly belongs to the same package level.